docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -67,20 +90,21 @@ image::images/graphical-view.png[]
 
 To see more details about each related process, select the process either in the left panel or the graphical view. The information on the process appears in the left panel, which typically includes but isn't limited to:
 
-- The number of events associated with the process.
-- The timestamp when the process was executed.
-- The file path of the process within the host.
-- The `process-pid`.
-- The user name and domain that ran the process.
-- Any other relevant process information. 
+* The number of events associated with the process.
+* The timestamp when the process was executed.
+* The file path of the process within the host.
+* The `process-pid`.
+* The user name and domain that ran the process.
+* Any other relevant process information. 
 
 [role="screenshot"]
 image::images/process-details.png[]
 
-See event details by selecting the **# events** URL at the top of the process details view or choosing one of the event pills in the graphical view.
+Loading process appear in a loading state when selecting that process for the first time. If loading data for a given process fails, simply click **Reload {process-name}** underneath the process to reload the data.
```

Comment:
```suggestion
When you first select a process, it appears in a loading state. If loading data for a given process fails, click **Reload `{process-name}`** beneath the process to reload the data.
```

Reply:
Adding a comment for lines 112-117 since I can't click the + sign due to the line number change. Let me know if the suggested change is hard to see. Also double-check `event.type` change to `event.category` - that's what it's named in the UI but I could be wrong. 

Original: 
`When selecting an `event.type`, a list of all those types of events appears in the left panel. If you want further details on the event, choose the event from the list. Event details appear in the left panel. 
[role="screenshot"]
image::event-details.png[]
In Elastic Security 7.10 and later, there is no limit to the number of events that can be associated with a process. However, in 7.9, each process is limited to only 100 events.`

```suggestion
When you select an `event.category`, pill, all the events within that category are listed in the left panel. To view more details about a specific event, select it from the list.
[role="screenshot"]
image::event-details.png[]
NOTE: In {stack} version >= 7.10.0, there is no limit to the number of events that can be associated with a process. However, in {stack} minor versions <= 7.9.0, each process is limited to only 100 events.
```